test/functional/test_site/expected/index.html

```diff
@@ -596,5 +673,18 @@ <h1 id="heading-in-footer-should-not-be-indexed">Heading in footer should not be
 <script src="markbind/js/setup.js"></script>
 <script src="SCRIPT_LINK"></script>
 <script>alert("hello")</script>
+<script src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
+<script>
+    function initAlgolia() {
+      docsearch({
+        apiKey: "TEST_API_KEY",
+        indexName: "TEST_INDEX_NAME",
+        inputSelector: "#algolia-search-input",
+        algoliaOptions: {},
+        debug: false,
+      });
+    }
+    MarkBind.afterSetup(initAlgolia);
```

Comment:
Not sure it's a good idea to make API calls to Agolia with invalid credentials when tests are run. Is there a way to test _only_ the `agolia-no-index` class _without_ starting the Agolia script?

Reply:
Unfortunately, I can't think of a way to execute only the `postRender` method without executing the rest of the plugin.

Alternatively, we can not do integration tests, and do only unit tests for this plugin - but I'm not sure if this is a good idea either.

Reply:
On further thought, we can use the demo `apiKey` and `indexName` provided in the DocSearch documentation.